### PR TITLE
#352  Rm npm step

### LINKED ISSUE #352: .drone.yml:16-17: Use ready image for drone npm step....
The puzzle `345-fd64d13a` from #345 has to be resolved:

https://github.com/fidals/shopelectro/blob/7a0860ec73f09921c59980111869af0d2319ad80/.drone.yml#L16-L17

The puzzle was created by duker33 on 15-Jun-18. 

Estimate: 60 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by duker33**
Closes #352

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -15,16 +19,19 @@ pipeline:
...
     environment:
       - DEPS_DIR=/drone/deps
     commands:
-      - npm install
-      - npm install -g gulp-cli
-      - gulp build
+      - cd docker
+      - cp drone_env/* env_files/
+      # Build python images with sources and static files
+      - docker-compose -f docker-compose-build.yml build --no-cache nodejs
```
Comment (artemiy312) [inline .drone.yml]: Let's  build nodejs in the `docker-build:` and move the pipe before the `test`

File: .drone.yml [hunk 2]
```diff
@@ -15,16 +19,19 @@ pipeline:
...
-    image: node:slim
+    image: docker/compose:1.17.1
     environment:
       - DEPS_DIR=/drone/deps
     commands:
-      - npm install
-      - npm install -g gulp-cli
-      - gulp build
+      - cd docker
+      - cp drone_env/* env_files/
```
Comment (artemiy312) [inline .drone.yml]: We do not use any env file for building of images

File: .drone.yml (hunk 1 above)
Comment (duker33) [reply .drone.yml]: @artemiy312 , it's for `gulp build` step below

Comment (duker33) [reply .drone.yml]: @artemiy312 , python-prod's build uses result of `dc run --rm nodejs gulp build`.
So, seems we are needed in separated npm step with `npm install` and `gup build` inside

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -35,9 +35,9 @@ services:
     image: fidals/se-nodejs:dev
     volumes:
       # Volumes for refarm-site's front development
-      - ../gulpfile.babel.js:/usr/app/src/gulpfile.babel.js
+#      - ../gulpfile.babel.js:/usr/app/src/gulpfile.babel.js
```
Comment (artemiy312) [inline docker/docker-compose.yml]: Let's undo changes from this file, seems they will broke `collectstatic`

Comment (duker33) [reply docker/docker-compose.yml]: sorry, forgot it

**Comment by duker33** [comment]
> It seems, you forgot to push commits

@artemiy312 , sorry, now pushed it

**Comment by duker33** [comment]
discussed code in a whole. Merge it to unlock new tasks

**Pull request merged**